components/portal/messages/controllers.js

```diff
@@ -205,13 +205,19 @@ define(['angular'], function(angular) {
             // Else hide messages features
             if (angular.equals($scope.$parent.showMessagesFeatures, true)) {
               configureNotificationsScope();
+              checkForChangesInOtherInstances;
             } else {
               vm.showMessagesFeatures = false;
               vm.isLoading = false;
             }
           }
         });
 
+        var checkForChangesInOtherInstances = $rootScope.$watch(
+          'priorityNotificationCount', function() {
+          configurePriorityNotificationsScope();
```

Comment:
:art: indent content within a function scope one level in from the function.

Reply:
Done